packages/shared/src/components/utilities/DateFormat.spec.tsx

```diff
@@ -0,0 +1,135 @@
+import { render, RenderResult, screen } from '@testing-library/react';
+import React from 'react';
+import DateFormat from './DateFormat';
+import { TimeFormatType } from '../../lib/dateFormat';
+
+const renderComponent = ({
+  date,
+  type,
+  className,
+  prefix,
+}: {
+  date: string | number | Date;
+  type: TimeFormatType;
+  className?: string;
+  prefix?: string;
+}): RenderResult => {
+  return render(
+    <DateFormat
+      date={date}
+      type={type}
+      className={className}
+      prefix={prefix}
+    />,
+  );
+};
+
+describe('date format post type', () => {
+  it('should render post time format', async () => {
+    renderComponent({
+      date: '2021-08-01T00:00:00Z',
+      type: TimeFormatType.Post,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Aug 01, 2021');
+  });
+
+  it('should render post time format as now', async () => {
+    const date = new Date();
+    renderComponent({ date, type: TimeFormatType.Post });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Now');
+  });
+
+  it('should render post time format as today', async () => {
+    const date = new Date();
+    renderComponent({
+      date: date.setHours(date.getHours() - 2),
+      type: TimeFormatType.Post,
+    });
+    const element = screen.getByRole('time');
+    expect(element).toHaveTextContent('Today');
```

Comment:
Should posts say `2 hours ago` as well?

Reply:
I believe our primary inspiration from this was YT, which mentions the hour as well.

![Screenshot 2024-03-07 at 7 10 44 PM](https://github.com/dailydotdev/apps/assets/13744167/e72ac339-6adc-4307-b03d-aa4147aec754)

Reply:
These only test for control values which have not been modified ;)

Reply:
Oh, right right. My bad, getting way ahead of myself. Too excited for this 😂